web/beacon-app/src/features/topics/components/TopicQueryResult/MetaDataTable.tsx

```diff
@@ -1,12 +1,21 @@
-import { t, Trans } from '@lingui/macro';
+import { Trans } from '@lingui/macro';
 import { Table } from '@rotational/beacon-core';
 import { ErrorBoundary } from '@sentry/react';
 
-const MetaDataTable = () => {
+import { getProjectQueryMetaData } from '../../utils';
+
+type MetaDataTableProps = {
+  results: any;
+};
+
+const MetaDataTable = ({ results }: MetaDataTableProps) => {
   const initialColumns: any = [
-    { Header: t`Key`, accessor: 'key' },
     {
-      Header: t`Value`,
+      Header: '',
```

Comment:
The headers have been removed to match the design. Also, the user's may not expect to see `Key` and `Value` here.